javascript/src/reducers/layers.js

```diff
@@ -3,8 +3,7 @@ import {ADD_VARIABLE, REMOVE_VARIABLE} from "../actions/variables"
 import {TOGGLE_VISIBILITY} from "../actions/map"
 import {FINISH_JOB} from "../actions/job"
 import {TOGGLE_LAYER} from '../actions/layers'
-import {RECEIVE_ZONES, RECEIVE_GEOMETRY} from "../actions/zones";
-
+import config from '../../../../javascript/src/seedsource/config'
```

Comment:
This import should be:

```javascript
import config from 'seedsource/config'
```

Reply:
Even as a relative import, this doesn't make sense. You would only need to do `../seedsource/config` since you're already in `javascript/src`.

Reply:
My thinking was there is a config in:

project-name/javascript/src/seedsource/config.js
Which I think of as my project config

And in:

../seedsource-core/javascript/src/seedsource/config.js
Which I think of as my seedsource config

The extra long import was to go one directory outside of seedsource config to project config. 
But looks like using ‘seedsource/config’ works too.

Reply:
But even if `seedsource/config` sisn't work, why `../../../../javascript/src/seedsource/config` instead of `../seedsource/config`?

Reply:
Oh, I get it! Never mind.

Reply:
And yes, `seedsource/config` is still the better one to use, because it doesn't presume the location of the application-specific config, which can be anywhere it wants.

Reply:
*in which case, seedsource/config will fall back to the seedsource-core config
Where is the logic for this written?

Reply:
In the webpack config at the application level.

E.g., https://github.com/consbio/csrt/blob/master/javascript/webpack.common.babel.js#L40-L41